src/app/portlets/dot-edit-page/layout/components/dot-edit-layout-grid/dot-edit-layout-grid.component.scss

```diff
@@ -4,10 +4,28 @@
 :host {
     display: block;
     width: 100%;
+
+    ::ng-deep {
+        dot-searchable-dropdown button,
+        dot-searchable-dropdown button:hover {
+            box-shadow: none;
+        }
+
+        dot-container-selector .container-selector__list {
+            height: 140px;
```

Comment:
What happen if the user adds so many containers? did start scroll?

Reply:
yes!